pkg/kn/commands/revision/delete.go

```diff
@@ -67,7 +107,28 @@ func NewRevisionDeleteCommand(p *commands.KnParams) *cobra.Command {
 			return nil
 		},
 	}
+	flags := RevisionDeleteCommand.Flags()
+	flags.Bool("prune", false, "Remove unreferenced revisions for a given service in a namespace.")
```

Comment:
this needs to be of type `flags.StringVar` for accepting the service name arg
see ref:
https://github.com/knative/client/blob/270e32240152cd02218932c72238edf0fd56dd04/pkg/kn/commands/revision/list.go#L112